config.yaml

```diff
@@ -1,307 +1,305 @@
-      requiredEntities: 
-       - name: "TransferStrategyInstalled"
-- id: 10
-  start_block: 0
-  contracts:
-  - name: OptimismAaveOracle
-    # abi_file_path: ./abis/AaveOracle.json
-    handler: ./src/EventHandlers.js
-    address:
-    - 0xD81eb3728a631871a7eBBaD631b5f424909f0c77
-    events:
-    - event: "AssetSourceUpdated(address indexed asset, address indexed source)"
-      requiredEntities: 
-       - name: "AssetSourceUpdated"
-    - event: "FallbackOracleUpdated(address indexed fallbackOracle)"
-      requiredEntities:
-       - name: "FallbackOracleUpdated"
-    - event: "BaseCurrencySet(address indexed baseCurrency, uint256 baseCurrencyUnit)"
-      requiredEntities: 
-       - name: "BaseCurrencySet" 
-  - name: OptimismPoolAddressesProviderRegistry
-    # abi_file_path: ./abis/PoolAddressesProviderRegistry.json
-    handler: ./src/EventHandlers.js
-    address:
-    - 0x770ef9f4fe897e59daCc474EF11238303F9552b6
-    events:
-    - event: "AddressesProviderRegistered(address indexed addressesProvider, uint256 indexed id)"
-      requiredEntities: 
-       - name: "AddressesProviderRegistered"
-    - event: "AddressesProviderUnregistered(address indexed addressesProvider, uint256 indexed id)"
-      requiredEntities: 
-       - name: "AddressesProviderUnregistered"
-    - event: "OwnershipTransferred(address indexed previousOwner, address indexed newOwner)"
-      requiredEntities: 
-       - name: "OwnershipTransferred"
-  - name: OptimismRewardsController
-    # abi_file_path: ./abis/RewardsController.json
-    handler: ./src/EventHandlers.js
-    address:
-    - 0x929EC64c34a17401F460460D4B9390518E5B473e
-    events:
-    - event: "AssetConfigUpdated(address indexed asset, address indexed reward, uint256 oldEmission, uint256 newEmission, uint256 oldDistributionEnd, uint256 newDistributionEnd, uint256 assetIndex)"
-      requiredEntities: 
-       - name: "AssetConfigUpdated"
-    - event: "Accrued(address indexed asset, address indexed reward, address indexed user, uint256 assetIndex, uint256 userIndex, uint256 rewardsAccrued)"
-      requiredEntities: 
-       - name: "Accrued"
-    - event: "RewardsClaimed(address indexed user, address indexed reward, address indexed to, address claimer, uint256 amount)"
-      requiredEntities: 
-       - name: "RewardsClaimed"
-    - event: "RewardOracleUpdated(address indexed reward, address indexed rewardOracle)"
-      requiredEntities: 
-       - name: "RewardOracleUpdated"
-    - event: "EmissionManagerUpdated(address indexed oldEmissionManager, address indexed newEmissionManager)"
-      requiredEntities: 
-       - name: "EmissionManagerUpdated"
-    - event: "ClaimerSet(address indexed user, address indexed claimer)"
-      requiredEntities: 
-       - name: "ClaimerSet"
-    - event: "TransferStrategyInstalled(address indexed reward, address indexed transferStrategy)" 
-      requiredEntities: 
-       - name: "TransferStrategyInstalled"
+  - id: 1
+    start_block: 16291000
```

Comment:
I looked at the contract on etherscan and too the start block from the earliest contract deployed on mainnet. Otherwise the indexer will start at block 0 and this will take a lot longer to search the blockchain for events